README.md

```diff
@@ -5,32 +5,32 @@ Simon Hughes
 
 20170329
 
-Version 1.00
+Version 0.1.0
```

Comment:
Anyways, as we have done a 1st public version already, we should start to claim it to be v1.0.0. 
Tag and version should match, that I 100% agree with.